src/components/App/EpisodesList/index.js

```diff
@@ -28,10 +28,10 @@ const renderEpisodesList = (episodes, func) => {
 const EpisodesList = props => {
   const { episodes, openModal } = props;
   return (
-    <div>
+    <>
```

Comment:
what is that? 
if you don't want to use wrapper you can render arraw of elements (React 16 feature)

Reply:
Please change it to `<React.Fragment>`

Reply:
OK. :)